src/tests/Login.test.jsx

```diff
@@ -107,3 +107,23 @@ it('handles wrong credentials', async () => {
     
     expect(screen.getByLabelText('errorMessage')).toHaveTextContent("The server is currently unavailable. Try again later")
   })
+
+it('input field', async () => {
+    render(<Login />)
+
+    fireEvent.change(screen.getByPlaceholderText('Username or Email'), {
+        target: { value: 'MyUsername' },
+    })
+    fireEvent.change(screen.getByPlaceholderText('Password'), {
+        target: { value: '12345678' },
```

Comment:
the same here, but you can also leave it as it is as the warning will be shown on the first empty field (so, if you test when username is empty, that'll suffice for the test purposes).